programs/ziti-edge-tunnel/netif_driver/linux/tun.c

```diff
@@ -130,12 +144,56 @@ static int run_command(const char *cmd, ...) {
     return r;
 }
 
-static int run_command_ex(bool log_nonzero_ec, const char *cmd, ...) {
-    va_list args;
-    va_start(args, cmd);
-    int r = run_command_va(log_nonzero_ec, cmd, args);
-    va_end(args);
-    return r;
+static int is_systemd_resolved_acquired_name(sd_bus *bus) {
+    int r;
+    _cleanup_(sd_bus_message_unrefp) sd_bus_message *reply = NULL;
+    _cleanup_(sd_bus_error_free) sd_bus_error error = SD_BUS_ERROR_NULL;
+    char** acquired = NULL;
+
+
+    r = sd_bus_call_method(
+            bus,
+            "org.freedesktop.DBus",
+            "/org/freedesktop/DBUS",
+            "org.freedesktop.DBus",
+            "ListNames",
+            &error,
+            &reply,
+            NULL);
+
+    if (r < 0) {
+        ZITI_LOG(ERROR, "Could not retreive DBus bus names: %s", error.message);
```

Comment:
`DBus` used here `dbus` used elsewhere. can we just normalize those one way or the other?